tssc/step_implementers/tag_source/git.py

```diff
@@ -1,5 +1,56 @@
-"""
-Step Implementer for the tag-source step for Git.
+"""Step Implementer for the tag-source step for Git.
+
+Step Configuration
+------------------
+
+Step configuration expected as input to this step.
+Could come from either configuration file or
+from runtime configuration.
+
+| Configuration Key | Required?          | Default              | Description
+|-------------------|--------------------|----------------------|-----------
+| `url`             | False              | git url returned by  | This is the url for the git
```

Comment:
What's the real-world use case where this won't be required? Shouldn't the TSC opinion be that this is required?

Reply:
we actually don't even use this param in our reference.

in the tag_source init doc it spells it out:

```
URL to the source repository to tag. Default to current directory
                                  source repository URL if not specified.
```

in the git impliementer its spelled out by what the default is. 99 times out of 100 likley just use the default. I think @kfrankli was just being overly cautios when he allowed the URL to be specified.